sdk/cosmosdb/cosmos/src/common/constants.ts

```diff
@@ -265,3 +267,68 @@ export enum OperationType {
   Execute = "execute",
   Batch = "batch"
 }
+
+/**
+ * @hidden
+ */
+export enum CosmosKeyType {
```

Comment:
I don't quite understand why this is needed. Does the user need to know which key they are using to create the SAS token? Why not just let them provide any key?